### DISCUSSION TITLE
Failed to signup

### DISCUSSION THREAD

**Discussion by jaanttil** [getfider/fider#1010] [Q&A]
Hi,

I was about to test Fider in Heroku. Followed instructions at https://getfider.com/docs/hosting-on-heroku and managed to get into sign-up screen and also received email verification/activation link. But when I click the link, the verification fails and I'm redirected to 404 with error "The link you clicked may be broken or the page may have been removed."'

Tested with main and v0.19.1. Any ideas what could be wrong? Heroku logs are not helpful either..

In _email_verifications_ table, there is one row and verified_at timestamp is set but user_id is empty. _users_ table has one row with admin role (3) and active status (1). 

Sign-up with another email as normal user works just find. If I try to sign-up again with initial/admin email, I receive link but it gives "The link you clicked has expired"

Thanks for a great tool & you help!

Janne

**Comment by goenning** [comment]
Is this related https://github.com/getfider/fider/issues/1007 ?

Are you using Microsoft Outlook to open the email?
